PixiEditor/Views/UserControls/PreviewWindow.xaml.cs

```diff
@@ -0,0 +1,113 @@
+﻿using PixiEditor.Models.DataHolders;
+using PixiEditor.Models.ImageManipulation;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace PixiEditor.Views.UserControls
+{
+    /// <summary>
+    /// Interaction logic for PreviewWindow.xaml
+    /// </summary>
+    public partial class PreviewWindow : UserControl
+    {
+        public static readonly DependencyProperty DocumentProperty =
+            DependencyProperty.Register(nameof(Document), typeof(Document), typeof(PreviewWindow));
+
+        public Document Document
+        { 
+            get => (Document)GetValue(DocumentProperty);
+            set => SetValue(DocumentProperty, value);
+        }
+
+        public static readonly DependencyProperty ColorCursorPositionProperty =
+            DependencyProperty.Register(nameof(ColorCursorPosition), typeof(Thickness), typeof(PreviewWindow));
+
+        public Thickness ColorCursorPosition
+        {
+            get => (Thickness)GetValue(ColorCursorPositionProperty);
+            private set => SetValue(ColorCursorPositionProperty, value);
+        }
+
+        public static readonly DependencyProperty ColorCursorColorProperty =
+            DependencyProperty.Register(nameof(ColorCursorColor), typeof(Color), typeof(PreviewWindow));
+
+        public Color ColorCursorColor
+        {
+            get => (Color)GetValue(ColorCursorColorProperty);
+            set => SetValue(ColorCursorColorProperty, value);
+        }
+
+        public static readonly DependencyProperty PrimaryColorProperty =
+            DependencyProperty.Register(nameof(PrimaryColor), typeof(Color), typeof(PreviewWindow));
+
+        public Color PrimaryColor
+        {
+            get => (Color)GetValue(PrimaryColorProperty);
+            set => SetValue(PrimaryColorProperty, value);
+        }
+
+        public PreviewWindow()
+        {
+            InitializeComponent();
+
+            MouseMove += PreviewWindow_MouseMove;
+            MouseRightButtonDown += PreviewWindow_MouseRightButtonDown;
+        }
+
+        private void PreviewWindow_MouseRightButtonDown(object sender, MouseButtonEventArgs e)
+        {
+            if (Keyboard.IsKeyDown(Key.LeftShift))
```

Comment:
Would be great to display this something to users.

Reply:
I thought about that too but I don't know where

Reply:
What about a small "?" icon button with a tooltip (make tooltips dark with rounded corners and maybe if possible small triangle on the bottom pointing down. Top edge connected to tooltip)

Reply:
+1 for that